05_DataQuality_functions.R

```diff
@@ -12,14 +12,15 @@ dq_main_reactive <- reactive({
   ServicesOnly <- calculate_long_stayers(input$ServicesOnlyLongStayers, 6)
   
   #Calculating potential old referrals based on Local settings
-  CE_Event <- calculate_outstanding_referrals(input$CEOutstandingReferrals) %>%
-    select(all_of(vars_we_want))
+  # CE_Event <- calculate_outstanding_referrals(input$CEOutstandingReferrals) %>%
```

Comment:
will likely need to uncomment this out once we get the specs with the rolled back CE elements

Reply:
will do in next feature branch. VL created us a card for it